### DISCUSSION TITLE
Is there a way to make the executable hide the console?

### DISCUSSION THREAD

**Discussion by Ripcord227** [marcelotduarte/cx_Freeze#1495] [General]
Is there a way to make it so the executable hides the console when ran similar to a .pyw file? I know autopy2exe does but I didn't see that in this.

**Comment by marcelotduarte** [comment]
In windows, yes. Use `base = "Win32GUI"`

https://cx-freeze.readthedocs.io/en/6.10/setup_script.html#cx-freeze-executable
